test_goldens/cupertino/cupertino_popover_menu_test.dart

```diff
@@ -405,11 +405,11 @@ Future<void> _pumpIosToolbar(
 }
 
 Future<void> _pumpToolbarScaffold(WidgetTester tester, {required Widget child}) async {
-  tester.binding.window
-    ..physicalSizeTestValue = const Size(500, 500)
+  tester.view
+    ..physicalSize = const Size(500, 500)
     ..platformDispatcher.textScaleFactorTestValue = 1.0
-    ..devicePixelRatioTestValue = 1.0;
-  addTearDown(() => tester.binding.window.clearAllTestValues());
+    ..devicePixelRatio = 1.0;
+  addTearDown(() => tester.view.reset());
```

Comment:
Should we use `addTearDown(() => tester.platformDispatcher.clearAllTestValues());` here?

Reply:
I don't know, should we? What's the difference?

Reply:
I'm not sure if `tester.view.reset()` will reset the `devicePixelRatio`.